tests/safeds/data/tabular/containers/_table/test_init.py

```diff
@@ -8,7 +8,7 @@
     ("table", "expected"),
     [
         (Table(), Schema({})),
-        (Table({}), Schema({})),
+        (Table(), Schema({})),
```

Comment:
The IDs suggest that this test is indeed supposed to distinguish between both cases. (Or rather, verify that both create the same table.)